src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  testError: string,
+  typeError: string,
+  stateError: boolean,
```

Comment:
You can pass something like this

`
type Props = {
  type: 'error' | 'success',
  text: string,
  isHidden: boolean
}
`